recipes/seqan3/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "SeqAn3" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/seqan/{{ name|lower }}/releases/download/{{ version }}/seqan-3.0.0-with-submodules.tar.gz
+  sha256: ba8f0fd5d19289f95823235b69fdc5a12aba3578292463eecf750722e8662bd3
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
```

Comment:
Can you add `zlib` and `bzip2` ?

Reply:
I believe this is unnecessary now (definitely for `zlib` but probably for `bzip2`), as they are always included

Reply:
That's cool, I thought they were just automatically pinned now.